### PR TITLE
Update Project Information

### PR DESCRIPTION
**Pull request by Kicchu02**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,36 +1,27 @@
...
     {
-      "name": "Amila Welihinda",
-      "email": "amilajack@gmail.com",
-      "url": "https://github.com/amilajack"
+      "name": "Narayana Murthy. B",
+      "email": "kicchu.2002@gmail.com",
+      "url": "https://github.com/Kicchu02"
+    },
+    {
+      "name": "Maithry V. Pappu",
```
Comment (law-levi-san) [inline package.json]: pls remove the period after V in my name, unless there's rules regarding nomenclature of contributors

File: package.json [hunk 2]
```diff
@@ -1,36 +1,27 @@
...
-    "hot",
-    "reload"
-  ],
-  "homepage": "https://github.com/electron-react-boilerplate/electron-react-boilerplate#readme",
-  "bugs": {
-    "url": "https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues"
-  },
+  "name": "draft-craft",
+  "description": "A markdown supported notes app.",
+  "keywords": [],
```
Comment (law-levi-san) [inline package.json:4]: what does this keywords array include? Will be populated in the future ?

File: package.json [hunk 3]
```diff
@@ -178,8 +169,7 @@
     "webpack-merge": "^5.9.0"
   },
   "build": {
-    "productName": "ElectronReact",
-    "appId": "org.erb.ElectronReact",
```
Comment (law-levi-san) [inline package.json:182]: what does this appId do? why was this removed?

File: package.json [hunk 4]
```diff
@@ -1,36 +1,27 @@
...
-    "sass",
-    "webpack",
-    "hot",
-    "reload"
-  ],
-  "homepage": "https://github.com/electron-react-boilerplate/electron-react-boilerplate#readme",
-  "bugs": {
-    "url": "https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues"
-  },
+  "name": "draft-craft",
```
Comment (law-levi-san) [inline package.json:2]: I believe naming it 'Draft-Craft' would be better, it will ensure uniformity across the app.

**Review by law-levi-san — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Kicchu02) [reply package.json:4]: This is an array of keywords will be used when someone is searching for certain things in the npm packages repository. If I include a keyword called `notes` and then upload this project, then if someone searches for `notes`, this project will also be shown in the search result. They are more like tag words. As of now, we needn't bother about it.

File: package.json (hunk 3 above)
Comment (Kicchu02) [reply package.json:182]: This `appId` is uniquely identifies our app in the world. It is usually the company website in reverse appended by the app name. This is used when we deploy it to any software store like play store, Microsoft store etc. As of now we are not bothered about distribution, and hence removed it. We can add it later when we need to distribute.

File: package.json (hunk 4 above)
Comment (Kicchu02) [reply package.json:2]: The regex for the `name` property doesn't allow for that casing. Hence, was forced to use lower case here.

File: package.json (hunk 2 above)
Comment (law-levi-san) [reply package.json:4]: > This is an array of keywords will be used when someone is searching for certain things in the npm packages repository. If I include a keyword called `notes` and then upload this project, then if someone searches for `notes`, this project will also be shown in the search result. They are more like tag words. As of now, we needn't bother about it.

oh alright

File: package.json (hunk 3 above)
Comment (law-levi-san) [reply package.json:182]: > This `appId` is uniquely identifies our app in the world. It is usually the company website in reverse appended by the app name. This is used when we deploy it to any software store like play store, Microsoft store etc. As of now we are not bothered about distribution, and hence removed it. We can add it later when we need to distribute.

cool

File: package.json (hunk 4 above)
Comment (law-levi-san) [reply package.json:2]: > The regex for the `name` property doesn't allow for that casing. Hence, was forced to use lower case here.

alright

**Review by law-levi-san — APPROVED** [review]
